src/components/EntityInfo.vue

```diff
@@ -149,7 +159,6 @@
                   v-if="isAllowed(AllowableActions.EDIT_BUSINESS_PROFILE)"
                   small text color="primary"
                   id="change-email-button"
-                  :disabled="hasBlocker"
```

Comment:
This is now handled by `isAllowed()` above. Basically the buttons don't display (instead of displaying disabled). I saw this here: https://projects.invisionapp.com/share/4U1217Y2S8TC#/screens/461385636